src/bucket/BucketBase.h

```diff
@@ -65,6 +65,9 @@ class BucketBase : public NonMovableOrCopyable
   public:
     static constexpr ProtocolVersion
         FIRST_PROTOCOL_SUPPORTING_PERSISTENT_EVICTION = ProtocolVersion::V_23;
+    static constexpr ProtocolVersion
+        FIRST_PROTOCOL_SUPPORTING_INITENTRY_AND_METAENTRY =
```

Comment:
This shouldn't be redefined here. It's only relevant for `LiveBucket`, so we should define it in `LiveBucket` only (which it currently is). It's static so we can use it everywhere via `LiveBucket::FIRST_PROTOCOL_SUPPORTING_INITENTRY_AND_METAENTRY`.